package/debian/test-package.sh

```diff
@@ -15,8 +15,12 @@ cd iele-semantics
 git checkout "$KIELE_REVISION"
 git submodule update --init --recursive
 
-iele-vm 0 127.0.0.1 > port &
+make test-vm -j4
+make test-iele -j4
+
+kiele vm --port 9001 &
```

Comment:
I would really much rather we kept things with a dynamically allocated port rather than a statically allocated port. This is a step backwards because the build will crash if this port is already busy.

Reply:
Not sure why the port would already be busy in the container on CI where this runs, but I can make the change.

But FYI, we've used statically determined ports for ages on teh Firefly build and that has never caused problems.

Reply:
It probably doesn't matter if it's just CI, but it needs to be able to work with a dynamically allocated port locally at the very least.